PureBasicIDE/Makefile

```diff
@@ -188,14 +188,23 @@ Build/DefaultTheme.zip: data/DefaultTheme/*
 	rm -f Build/DefaultTheme.zip
 	zip -jq Build/DefaultTheme.zip data/DefaultTheme/*
 
-# Use "mkdir" instead on mkdir, so on Windows it search mkdir.exe on the PATH instead of using the internal CMD mkdir.
-# It works on Linux and OS X as well.
+# On Linux and OS X, use "mkdir" (mkdir.exe) instead of mkdir, to make it works
+# On Windows, use internal mkdir command, mkdir.exe does not exist and internal mkdir does not support -p switch
+# %OS% is the type of windows: Windows_NT
 #
+ifeq ($(OS),Windows_NT)
 $(THEMEFOLDER)/SilkTheme.zip: data/SilkTheme/*
-	"mkdir" -p "$(THEMEFOLDER)"
+	mkdir "$(THEMEFOLDER)"
 	rm -f "$(THEMEFOLDER)/SilkTheme.zip"
 	zip -jq "$(THEMEFOLDER)/SilkTheme.zip" data/SilkTheme/*
 
+else
+$(THEMEFOLDER)/SilkTheme.zip: data/SilkTheme/*
+	"mkdir.exe" -p "$(THEMEFOLDER)"
+	rm -f "$(THEMEFOLDER)/SilkTheme.zip"
+	zip -jq "$(THEMEFOLDER)/SilkTheme.zip" data/SilkTheme/*
+endif
+
 $(COLORTABLE): data/ColorTable.xml
 	cp -f data/ColorTable.xml "$(COLORTABLE)"
 
```

Comment:
Never change the Makefile, it's used by the build system. May be I should put a comment on top of it. You should use the MakeWindow.cmd on Windows

Reply:
Oh sorry Fred, the goal was to help, not to waste your precious time.
It's probably best to delete this Pull Request and to leave it up to you to solve the 3 issues
Can you delete it please, I'm afraid I do worse than better on my Github's friend.

Thanks for the comments even if here:
On windows changing #COMPILER_FileFormat #PB_UTF8 to #PB_Ascii, looks good here to accept special chars (É). With UTF-8 the Linker ends in error if there are any special chars.
There are probably still a few tools (maybe vc/link.exe) that communicate using Ascii.

I understand for the makefile, I'll use the quick & easy method now: MakeWindows.cmd <YourPureBasicPath> .
But then, it's good to know that the official method doesn't fully work on Windows. "mkdir" -p is not recognized (mkdir.exe does not exist on windows) and so the Themes folder is not (re)created. At the same time, it's already there.

Finally, the only thing that seems ~ right is PeekAscii(*Debugger\CommandData) => PeekUTF8(*Debugger\CommandData) in IDEDebugger and StandaloneDebugger.pb
For special characters in error or warning messages in the debugger.

Reply:
Don't worry, no problem here. About mkdir, you need to put the unixtools path in your PATH to have the unix command instead of the Windows one.

I will patch the PeekUTF8 stuff, so I will close this request